src/globus_sdk/services/auth/client_types/base.py

```diff
@@ -69,8 +69,8 @@ def __init__(self, client_id: Optional[str] = None, **kwargs: Any) -> None:
 
     def get_identities(
         self,
-        usernames: Optional[Union[List[ToStr], ToStr]] = None,
-        ids: Optional[Union[List[ToStr], ToStr]] = None,
+        usernames: Optional[Union[Sequence[ToStr], ToStr]] = None,
+        ids: Optional[Union[Sequence[ToStr], ToStr]] = None,
```

Comment:
I think I'd prefer to see
```
usernames: Sequence[AnyStr] | AnyStr | None
ids: Sequence[UUIDLike] | UUIDLike | None
```
([typing.AnyStr](https://docs.python.org/3/library/typing.html#typing.AnyStr))

Otherwise, you can pass `ids=4` or `usernames=UUID(...)`, neither of which seem valuable to allow.

Reply:
Yes, these and some other current `ToStr` uses (especially in TransferClient) still need updating to use the more specific versions. I'm in the middle of that pass now.

Reply:
Sorry for the noise from the reviewing perspective, I was trying to link specific commits so you wouldn't have to troll through everything and sort out what still did or didn't need looking at.

Reply:
No worries at all. I'll wait to look until later. I'm just trying to get you feedback before I go on break!